starter-code/css/main.css

```diff
@@ -7,4 +7,99 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-color: #1c5380;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #1c5380;
+}
+
+#brand {
+  width: 50%;
+  height: 50%;
+}
+
+#home {
+  background-color: #1c5380;
```

Comment:
This is ok, but better than a plain color background, could have been a gradient color :)
https://www.w3schools.com/css/css3_gradients.asp